### PR TITLE
Added UI

### PR DESCRIPTION
**Pull request by justakit**
Added basic UI functionality

### CODE REVIEW

File: client/ui/uimain.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"os"
+
+	"github.com/therecipe/qt/core"
+	"github.com/therecipe/qt/quick"
+	"github.com/therecipe/qt/widgets"
+)
+
+func CreateWindow() {
+	widgets.NewQApplication(len(os.Args), os.Args)
+	var layout = widgets.NewQHBoxLayout()
```
Comment (8tomat8) [inline client/ui/uimain.go:14]: You can use short syntax for simplicity.
For example:
`layout := widgets.NewQHBoxLayout()`

File: client/main.go [hunk 2]
```diff
@@ -1,7 +1,9 @@
 package main
 
-import "fmt"
+import (
+	"SSU-Golang-252-Chat/client/ui"
```
Comment (8tomat8) [inline client/main.go]: Import path should look like this `"github.com/8tomat8/SSU-Golang-252-Chat/client/ui"`

**Review by 8tomat8 — APPROVED** [review]
Looks good to me. Please look for some comments

File: client/ui/uimain.go (hunk 1 above)
Comment (justakit) [reply client/ui/uimain.go:14]: If it possible to leave "**var** varName" for better highlighting of variables?

File: client/main.go (hunk 2 above)
Comment (justakit) [reply client/main.go]: Import path changed

**Pull request merged**